hedera-node/test-clients/src/main/resource/testSystemFiles/zero-ops-group.json

```diff
@@ -7,36 +7,37 @@
 		{
 		  "opsPerSec": 0,
 		  "operations": [
-			"CryptoCreate",
-			"CryptoTransfer",
-			"UtilPrng",
-			"CryptoUpdate",
-			"CryptoDelete",
-			"CryptoGetInfo",
-			"CryptoGetAccountRecords",
-			"CryptoApproveAllowance",
-			"CryptoDeleteAllowance",
-			"ConsensusCreateTopic",
-			"ConsensusSubmitMessage",
-			"ConsensusUpdateTopic",
-			"ConsensusDeleteTopic",
-			"ConsensusGetTopicInfo",
-			"TokenGetInfo",
-			"TokenGetNftInfo",
-			"TokenGetAccountNftInfos",
-			"ScheduleDelete",
-			"ScheduleGetInfo",
-			"FileGetContents",
-			"FileGetInfo",
-			"ContractUpdate",
-			"ContractDelete",
-			"ContractGetInfo",
-			"ContractGetBytecode",
-			"ContractGetRecords",
-			"ContractCallLocal",
-			"TransactionGetRecord",
-			"GetVersionInfo"
-		  ]
+				"CryptoCreate",
+				"CryptoTransfer",
+				"UtilPrng",
+				"CryptoUpdate",
+				"CryptoDelete",
+				"CryptoGetInfo",
+				"CryptoGetAccountRecords",
+				"CryptoApproveAllowance",
+				"CryptoDeleteAllowance",
+				"ConsensusCreateTopic",
+				"ConsensusSubmitMessage",
+				"ConsensusUpdateTopic",
+				"ConsensusDeleteTopic",
+				"ConsensusGetTopicInfo",
+				"TokenGetInfo",
+				"TokenGetNftInfo",
+				"TokenGetNftInfos",
```

Comment:
Added that because the test that uses this file is testing for opsPerSec=0 but the file had two problems: opsPerSec=0 and missing expected operation. So in order to make the test test only single thing - I fix the second problem